docs/release/release-tasks.md

```diff
@@ -25,6 +25,7 @@ This document details the responsibilities and tasks for each role in the releas
       - [\[Track\] Bump dependencies](#track-bump-dependencies)
       - [Create a release branch](#create-a-release-branch)
       - [\[Continuously\] Maintain the GitHub release milestone](#continuously-maintain-the-github-release-milestone)
+      - [\[Continuously\] Bump the Go version](#continuously-bump-the-go-version)
```

Comment:
I think this fits well into similar tasks like "Bump the Kubernetes version" which is why I added it to release lead.

No strong opinion from my side, if we think it fits better somewhere else.

I think overall it makes sense that the release team is responsible for keeping the Go version up-to-date.

Opinions?

Reply:
What about phrasing in a way that the release manager is responsible for beating the drum, while someone else can do the work? with this regards his responsibility is to open issues, ask for volunteers at the office hours, monitor completion

Reply:
I can do it, but it seems like a lot of work considering:
* those tasks are usually already shared across release lead and shadows
* bumping the Go version comes usually down to changing two characters

Reply:
It looks fine here. The release lead can be responsible for this. I might be upto the person to either do it themselves or get some volunteers involved (even though it might not be a lot of work just for the sake of spreading the knowledege).

How about using the `[Track]` prefix to be inline with items like "Bump Dependencies". This way it also hints that anyone can actually do the work.
No strong opinions.

Reply:
Technically every task can be done by anyone/ delegated. Where do we draw the line?

Reply:
ok given that we all agree that the release lead can/should delegate.